resources/public/api/conf/1.0/retrieve_se_bsas.raml

```diff
@@ -0,0 +1,26 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
```

Comment:
Probably just the spec being inconsistent, but the spec says `The format of the supplied National Insurance number is invalid.` - which is correct?